addon/PlacesProvider.js

```diff
@@ -643,6 +643,28 @@ Links.prototype = {
   }),
 
   /**
+   * Gets History size since a certain timestamp
+   *
+   * @param {String} timestamp in milliseconds
+   *
+   * @returns {Promise} Returns a promise with the count of moz_places records
+   *                    that have been entered since the timestamp provided
+   */
+  getHistorySizeSince: Task.async(function*(timestamp) {
+    if (!timestamp) {
```

Comment:
This if statement will be unnecessary if `asyncGetOldestInsert` either returns a valid timestamp, or raises an exception.